### ISSUE TITLE
Accessibility issue on editing text when accessing through JAWS screen reader

### ISSUE THREAD

**Issue by yerrinaidu** [tinymce/tinymce#6129]
Can’t input any text if I turn in the jaws screen reader 

Turn the screen reader on and try to type something in the editor, it wouldn’t let you type 
**What is the expected behavior?**
It should Input the text with jaws screen on 
**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

**Comment by lnewson** [comment]
Hi @yerrinaidu, I attempted to reproduce this using JAWS 2020 on the latest versions of IE 11, Chrome and Firefox and I was unable to reproduce this issue. Can you please include more details on how to reproduce this issue and also what browser you're able to reproduce it on?

**Comment by yerrinaidu** [comment]
Hi Lee,
Thanks for getting back to me so quickly, I was using JAWS 2019 version,
and testing against chrome browser. I couldn’t enter any text if I turn the
JAWS, but interestingly i could enter text if I press Insert + z .

Ps: it used work fine 5 months back, I am seeing this issue recently with
both 2018 and 2019 versions of JAWS. Haven’t tried with 2020 version yet.

**Comment by thalesassis** [comment]
I had this issue yesterday, using TinyMCE 4.1 and Jaws 2020 latest version. It doesn't work with Virtual Cursor ON (if you use Insert + Z to deactivate, then you can input text). The solution was updating TinyMCE to 4.9.

**Comment by lnewson** [comment]
Sorry for the delayed reply, but I tested this and JAWS 2019 appeared to be working on all supported browsers/TinyMCE versions. I was able to reproduce the issue on legacy edge, but it's no longer supported and appeared to be a JAWS bug. So, given all the information it sounds more likely to be a JAWS bug than a TinyMCE bug.

However, if you can provide a reproduction case on a supported version we'll be happy to look into it more.
